ansible/LibInventory.pm

```diff
@@ -4,8 +4,9 @@
 #
 use strict;
 
-#use JSON;
+use JSON;
```

Comment:
Does this PR supersede https://github.com/socallinuxexpo/scale-network/pull/619?

Reply:
I don't think so... I think it's orthogonal.

Reply:
It's not unlikely, however, that the changes in 619 got included here due to my poor management of branches.